packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/address/index.js

```diff
@@ -1,10 +1,43 @@
 const express = require('express');
 const { createSubmissionAddress, deleteSubmissionAddress } = require('./controller');
+const { AUTH } = require('@pins/common/src/constants');
+const config = require('../../../../../../configuration/config');
+const { auth } = require('express-oauth2-jwt-bearer');
+const { validateToken } = require('@pins/common/src/middleware/validate-token');
+const { openApiValidatorMiddleware } = require('../../../../../../validators/validate-open-api');
+const lpaCanModifyCase = require('../lpa-can-modify-case');
 const asyncHandler = require('@pins/common/src/middleware/async-handler');
+
 const router = express.Router({ mergeParams: true });
 
-// Document upload routes
-router.post('/', asyncHandler(createSubmissionAddress));
-router.delete('/:addressId', asyncHandler(deleteSubmissionAddress));
+router.use(
```

Comment:
DW about it in this PR but we've got this in a few places now. Maybe we make an authedRouter that has this middleware already on it?

Reply:
that would make a lot of sense